### ISSUE TITLE
[WeeklyReports] 2026.3.23~2026.4.12 周报汇总

### ISSUE THREAD

**Issue by fangfangssj** [PFCCLab/Camp#566]
# [WeeklyReports] 2026.3.23~2026.4.12 周报汇总

请各位学员在本 issue 下以 comment 的形式填写周报摘要，ddl 下周一晚，格式示例如下：

```
### 姓名

xxx

### 本周工作

1. xxx
2. xxx
 
### 下周工作

1. xxx
2. xxx

### 详细周报链接：

- https://github.com/PFCCLab/Camp/pull/xxx

```

#### 项目一：[PaddleOCR + ERNIE × Open-Source Ecosystem 高价值开源项目案例征集]

- [x] @LiaoYFBH (@openvino-book)

#### 项目二：[Paddle API兼容性增强]

- [x] @algorithm1832 (@zhwesky2010)
- [x] @Manfredss (@zhwesky2010)

#### 项目四：[Paddle C++ API生态兼容建设]

- [x] @youge325 (@BingooYang)

#### 项目七：[FlashAttention 低精度训练算法研究与 Kernel 开发]

- [x] @ArcaLunar (@GuoxiaWang)

#### 项目一：[基于AI Coding的飞桨-Torch编译能力融合探索]

- [ ] @CJ77Qi (@SigureMo)

#### 项目二：[Paddle API兼容性增强]

- [x] @Le-soleile (@zhwesky2010)

#### 项目三：[PaConvert转换工具建设+官网映射文档建设]

- [x] @Aidenwu0209 (@zhwesky2010)

#### 项目四：[面向大模型训练的高效分布式checkpoint系统研究]
- [ ] @kkaaaaaaa (@xingminyyj)

**Comment by algorithm1832** [comment]
### 姓名
杨锃砚

### 本周工作

1. [Used AI Studio] `paddle.get_rng_state`添加别名
2. [Used AI Studio] `paddle.Tensor.numpy()`添加参数支持
3. [Used AI Studio] `paddle.autograd.function.once_differentiable`添加别名
4. [Used AI Studio] `PyLayerContext.saved_tensors`添加别名
5. [Used AI Studio] `paddle.Tensor.fill_diagonal_`加装饰器
6. [Used AI Studio] `paddle.optimizer.optimizer.step`添加参数
7. [Used AI Studio] `paddle.Tensor.type`支持调用
8. 补充英文文档
9. 修复API别名测试
 
### 下周工作

1. 修改PaConvert和文档
2. 完善`paddle.Tensor.type`的修改方案并实现

### 详细周报链接：
#570

**Comment by Aidenwu0209** [comment]
### 姓名

Aidenwu0209

### 本周工作

1. 补充 `paddle.nn.functional.binary_cross_entropy` 中文文档中的 `target` alias 说明，完成 docs PR [#7887](https://github.com/PaddlePaddle/docs/pull/7887) 并合入。
2. 补充 `paddle.rot90` 参数别名兼容文档，完成 docs PR [#7826](https://github.com/PaddlePaddle/docs/pull/7826) 并合入。
3. 重构 PaConvert CI 脚本职责，完成 code set convert 与 consistency check 的解耦，提交并合入 PR [#865](https://github.com/PaddlePaddle/PaConvert/pull/865)。
4. 针对 UnitTest 首轮失败、重跑通过的 flaky 场景进行治理，提交 PR [#867](https://github.com/PaddlePaddle/PaConvert/pull/867)，当前仍在推进中。
 
### 下周工作

1. 继续推进 PaConvert UnitTest CI 稳定性优化，跟进 PR [#867](https://github.com/PaddlePaddle/PaConvert/pull/867) 的评审与问题修复。
2. 继续梳理 PaConvert CI 脚本与测试流程，定位并收敛剩余 flaky 问题。

### 详细周报链接：

- https://github.com/PaddlePaddle/docs/pull/7887
- https://github.com/PaddlePaddle/docs/pull/7826
- https://github.com/PaddlePaddle/PaConvert/pull/865
- https://github.com/PaddlePaddle/PaConvert/pull/867

**Comment by youge325** [comment]
### 姓名

谢润明

### 实习项目

Paddle C++ API生态兼容建设

### 本周工作

1. 回滚 DeepGEMM 和 DeepEP 当中的部分修改以减少 diff

2. 在 Paddle 仓库中对齐部分接口

3. 在 PaddleFleet 仓库中更新 DeepEP

4. 更新 PaddleCppAPITest 仓库下的文档及测试

5. 在 PaddleCodec 中新增 ARM 设备 CI


### 下周工作

1. 跟进未合入的 pr

3. 尝试新增 paddlecodec 上游缺失的兼容接口，进行排查后发现，目前缺失较多


### 详细周报链接：

- https://github.com/PFCCLab/Camp/pull/576

**Comment by LiaoYFBH** [comment]
### 姓名

LiaoYFBH

### 实习项目

[PaddleOCR + ERNIE × Open-Source Ecosystem 高价值开源项目案例征集]

### 本周工作

1. 在端到端科研Agent流程中，增加了基于LLM的评审模型（Reviewer）。
2. 增加了PDF生成模块。
3. 调研了可用于科研Agent评测的benchmark，初步集成了ScienceAgentBench（102个科学计算任务），后续还将考虑MLAgentBench、GAIA等benchmark。

### 下周工作

1. 修复评审模型未生效的问题，使评审结果能实际引导agent修正输出。
2. 增加终端交互模式，支持不依赖Gradio UI直接在命令行运行和交互。
3. 在ScienceAgentBench等benchmark上进行实际评测，收集性能指标。
4. 找一些用户进行实际体验，收集反馈建议。

### 详细周报链接：
https://github.com/PFCCLab/Camp/pull/575

**Comment by Le-soleile** [comment]
### 姓名

杨晓春

### 实习项目

Paddle API兼容性增强

### 本周工作

1. 学习装饰器设计、`C++`下沉技术文档、学习`Paddle API`五层架构实现
2. 新增`tile`专有装饰器：https://github.com/PaddlePaddle/Paddle/pull/78516
3. 下沉`exp_ `：https://github.com/PaddlePaddle/Paddle/pull/78617
 
### 下周计划

1. 下沉`python/paddle/tensor/ops.py`中所有`inplace unary API`
2. 继续学习`Paddle API`实现源码，主要看`Pybind、Dygraph、C++ API`三层自动生成逻辑（`.yaml`文件每个字段影响的文件、生成函数内部运作逻辑）

### 详细周报链接
[577](https://github.com/PFCCLab/Camp/pull/577)

**Comment by ArcaLunar** [comment]
### 姓名

徐启越

### 实习项目

FlashAttention 低精度训练算法研究与 Kernel 开发

### 本周工作

1. 在已有 NCU / NSYS profiling 结果基础上，对 kernel 性能进行了进一步拆解分析，重点关注内存访存模式、SM 利用率、warp stall 原因．

2. 经过大量测试，包括大矩阵、特殊 mask，新实现在一些数据测试上出现与 upstream baseline 产生误差的现象．正在排查原因，包括实现路径上的类型转换，累加器类型差异等等

3. 基于 profiling 结果，对后续优化路径进行了系统梳理与调研学习，包括访存优化模式、并行策略．

### 下周计划

1. 针对 kernel profiling 检测出来的特征做进一步优化
2. 排查出现误差的原因，并基于排查结果选择在模型上测试或是修改实现
3. 调研并学习其他优化路径

### 详细周报链接

https://github.com/PFCCLab/Camp/pull/582

**Comment by Manfredss** [comment]
### 姓名
戚文斐

### 本周工作
#### 已合并
1. [PR #78342](https://github.com/PaddlePaddle/Paddle/pull/78342) 新增 `paddle._assert` api
2. [PR #78301](https://github.com/PaddlePaddle/Paddle/pull/78301) 对齐 `paddle.nn.Layer.to` 参数顺序
3. [PR #78298](https://github.com/PaddlePaddle/Paddle/pull/78298) 对齐 `paddle.logit`, 添加参数别名 `input`; 新增参数 `out`; 在 `paddle.special` 中添加注册

#### 进行中
1. [PR #7893](https://github.com/PaddlePaddle/docs/pull/7893) 为之前所有新增或对齐的 api 更新中文文档 
2. [PR #78441](https://github.com/PaddlePaddle/Paddle/pull/78441) 在 AI Studio 开发机上测试覆盖率问题
3. [PR #77333](https://github.com/PaddlePaddle/Paddle/pull/78441) 在 AI Studio 开发机上测试 PR 问题
4. [PR #78473](https://github.com/PaddlePaddle/Paddle/pull/78473) 为 `conv_transpose` 添加装饰器，暂时解决参数顺序不一致的问题

### 下周工作
1. 着重修复 `aminmax` 和 `addcmul` 两个新 api 的问题


### 详细周报连接
- https://github.com/PFCCLab/Camp/pull/583
